### ISSUE TITLE
Password field doesn't accept passphrases (passwords with spaces)

### ISSUE THREAD

**Issue by keval6b** [goharbor/harbor#15930]
If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**
- Tried to change password to (for example): `test PASSWORD 123`
- Error:
> Password should be 8-20 characters long with at least 1 uppercase, 1 lowercase and 1 number.

![image](https://user-images.githubusercontent.com/33595996/139948942-63efae24-ccf5-476e-be8c-373f9d0a9618.png)
- Expected behavior: Sets password

**Steps to reproduce the problem:**
- Fresh install of harbor with all defaults minus https
- Edit password of admin user

**Versions:**
- harbor version: [2.4.0]
- docker engine version: [20.10.10]
- docker-compose version: [2.0.1]

**Comment by wy65701436** [comment]
yes, it's by designed. Can you give more background about this requirement?

**Comment by thechubbypanda** [comment]
Passphrases are becoming more common since they are easier to remember, almost all other applications support them.

**Comment by nihcosnosaj** [comment]
Hi, can I take a shot at implementing this?

**Comment by qnetter** [comment]
> Hi, can I take a shot at implementing this?

Yes please.
